web/client-api/src/main/java/io/deephaven/web/client/ide/IdeClient.java

```diff
@@ -19,13 +19,10 @@ public Promise<IdeSession> getExistingSession_old(String websocketUrl, String au
         return IdeClient.getExistingSession(websocketUrl, authToken, serviceId, language);
     }
 
+    @Deprecated
```

Comment:
Should mark as deprecated in WebDevelopersGuide as well.

Reply:
I'm holding off in favor of moving this all to type script in the next week or two, then deleting the .md file?

Reply:
Okay